BuildLoader.py

```diff
@@ -264,6 +264,7 @@ def __init__(self, *args, **kwargs):
         self.BUILD_ARCH            = ''
         self.KEYH_SVN              = 0
         self.CFGDATA_SVN           = 0
+        self.BUILD_IDENTICAL_TS    = 0
```

Comment:
TS is always same for A and B. From its usage maybe we could change "BUILD_IDENTICAL_TS" to "REBASE_STAGE1B" to make this setting more clear?

Reply:
Right now, this flag is only to control the rebasing of SG1B. However, in an upcoming patch, this flag will also control the patching of the flash map with the boot partition. I just didn't add this change here because it also requires a *.c code change.